### PR TITLE
Delta elektronica power supply sm70-45D

### PR DESCRIPTION
**Pull request by CasperSchippers**
**Ready for merging**

Added support for the Delta Elektronika SM70-45D Power supply (serves also as an example for other power supplies of Delta Elektronika as they use the same communication)

### CODE REVIEW

File: pymeasure/instruments/deltaelektronika/sm7045d.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+    def enable(self):
+        # Disable remote shutdown, hence output will be enabled.
+        self.write("SO:FU:RSD 0")
+
+    def disable(self):
+        # Enables remote shutdown, hence input will be disabled.
+        self.write("SO:FU:RSD 1")
+
+    def ramp_to_current(self, target_current, current_step):
```
Comment (cjermain) [inline pymeasure/instruments/deltaelektronika/sm7045d.py]: I think it is worth setting a default for `current_step`.

Comment (CasperSchippers) [reply pymeasure/instruments/deltaelektronika/sm7045d.py]: That's a good suggestion, i implemented it

**Comment by cjermain** [comment]
Looks good. Thanks for improving the methods.

**Comment by mjgardes** [comment]
I think there is a typo in sm7045d.py at line 72

```
    max_current = Instrument.control(
        "SO:CU:MA?", "SO:VO:MA %g",
```

should probably be
```
    max_current = Instrument.control(
        "SO:CU:MA?", "SO:CU:MA %g",
```

**Comment by CasperSchippers** [comment]
I think you're right, @mjgardes. Thanks for the remark. I will include it in an open PR that (by now) includes a number of instrument fixes (#418)

**Pull request merged**